about/docs_changelog.rst

```diff
@@ -7,7 +7,7 @@ The documentation is continually being improved. The release of version 3.1
 includes many new tutorials, many fixes and updates for old tutorials, and many updates
 to the class reference. Below is a list of new tutorials added since version 3.0.
 
-.. note:: This document only contains new tutorials so not all changes are reflected, 
+.. note:: This document only contains new tutorials so not all changes are reflected,
           many tutorials have been substantially updated but are not reflected in this document.
 
 New tutorials since version 3.0
```

Comment:
We're working on 3.2 now, so there should be a "New tutorials since version 3.1" section for everything made after 3.1-stable.